chart/test/test.yaml

```
@@ -22,7 +22,7 @@ crawler_channels:
     image: "docker.io/webrecorder/browsertrix-crawler:latest"
 
   - id: test
-    image: "docker.io/webrecorder/browsertrix-crawler:1.1.0-beta.1"
+    image: "docker.io/webrecorder/browsertrix-crawler:1.1.0-beta.4"
```

```suggestion
    image: "docker.io/webrecorder/browsertrix-crawler:1.1.0-beta.5"
```

We just released 1.1.0-beta.5, though soon will be able to revert to latest as soon as 1.1.0 is released.